plugin/node.js

```diff
@@ -98,9 +98,14 @@
     } else if (name in data.modules) {
       result = data.modules[name];
     } else if (data.options.modules && data.options.modules.hasOwnProperty(name)) {
-      var scope = buildWrappingScope(cx.topScope, name);
-      infer.def.load(data.options.modules[name], scope);
-      result = data.modules[name] = scope.exports;
+      var mod = data.options.modules[name];
+      if (typeof(mod) == "string" && mod.charAt(0) == '$') {
+        result = cx.topScope.props[mod.substring(1)];
```

Comment:
This means you can only directly load globals. See [this part](https://github.com/marijnh/tern/blob/master/plugin/requirejs.js#L72) of the requirejs plugin, which uses `infer.def.parsePath` instead, so that it can refer to arbitrary paths. Also, it would be good to stick to the convention used in that plugin and use `=` as the leading character to denote references to the existing environment.